### ISSUE TITLE
Exhaustion doesn't work properly for movement (at least)

### ISSUE THREAD

**Issue by Kariaudhanaur** [DFreds/dfreds-convenient-effects#144]
**Describe the bug**
Not a very big deal, but exhaustion level are not parameterized the same way than it is in the tidy sheet, and doesn't work properly in DFreds (at least some)

**To Reproduce**
For example, exhaustion level 2, yours on the right :
 
![image](https://user-images.githubusercontent.com/92259502/167945309-9904b73a-0a9e-4c75-b6a3-7d532002b850.png)

I'm not sure if it's intended or not, but the thing that made me disturb you about that is the "move reduction" doesn't work properly :
data.attributes.movement.walk
for a barbarian who has extra move with level 5, only the base "9 meters / 30 feet" are divided, not the bonus... But directly from the tidysheet, it does (because of the "multiply" and not "customize" I guess ? )

This issue is just in case you don't know that already... Thanks for your module anyway. (can't wait for the localisation)

**Comment by DFreds** [comment]
Okay, thanks for the heads up. I'm guessing this is in dnd5e 1.6?

**Comment by Kariaudhanaur** [comment]
Yes absolutely... I admit I don't know if it was already the case before 1.6 ... 
I previously used your module only for conditions... I added exhaustion ("épuisement") recently...
